src/main/java/frc/robot/Constants.java

```diff
@@ -28,8 +22,9 @@ public static class Auto {
     public static final double k_maxAcceleration = 1.0; // NOT USED
```

Comment:
Has this ever been used? Because I don't recall it ever having a use.... Why do we have it?